### ISSUE TITLE
[Feature Request] Add support for Ollama

### ISSUE THREAD

**Issue by MohamedBassem** [MohamedBassem/hoarder-app#3]
This issue to track supporting ollama as an alternative to openai.

**Comment by dillfrescott** [comment]
Yes!!!!

**Comment by MohamedBassem** [comment]
Ollama support is now available in the `0.10.0` release! 🎉 You can check the new configuration arguments [here](https://docs.hoarder.app/configuration#inference-configs-for-automatic-tagging)!

Basically add the following to the env file (make sure you remove the `OPENAI_API_KEY` otherwise it takes precedence):

```
OLLAMA_BASE_URL=<ollama_addr>
INFERENCE_TEXT_MODEL=llama2
INFERENCE_IMAGE_MODEL=llava
```

Make sure that those two models are pulled in ollama with `ollama pull llama2` and `ollama pull llava`.

**Comment by dillfrescott** [comment]
Id recommend the openchat 0106 model for text, that thing is impressive for a 7b model!
